.github/workflows/finale.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {request} = require('gaxios');
+const args = process.argv.slice(2);
+const requiredJobs = JSON.parse(args[0]);
+const runId = process.env.GITHUB_RUN_ID;
+const repo = process.env.GITHUB_REPOSITORY;
+const githubToken = args[1];
+
+async function main() {
+  for (;;) {
```

Comment:
Is it possible that a check fails to report? It looks like the finale script will continue execute until the max execution time (if there is one)?

Reply:
Yeah, it would have been.  I set a (somewhat arbitrary) max execution time of 20 minutes. We can obviously play with it as we go.